test/ChatRoom.test.js

```diff
@@ -0,0 +1,52 @@
+const ChatRoom = require('../lib/ChatRoom');
+
+describe('ChatRoom', () => {
+
+  let chatroom = null;
```

Comment:
```suggestion
  let chatroom = null;
  let client1;
  let client2;

  beforeEach(() => {
    chatroom = new ChatRoom();
    client1 = {};
    client2 = {};
    chatroom.add(client1);
    chatroom.add(client2);
  });
```
Here is another way you could approach setting up the tests -- adding clients in a beforeEach allows you to eliminate the need to add a client in every test.